standards/source-code.md

```diff
@@ -0,0 +1,91 @@
+# Source code
+The default position for FFC is that code should be open source and hosted within GitHub.
+
+If there is a need for a private repository then an AzureDevOps Repo should be used instead.
+
+## Creating repositories
+### Naming repositories
+Repositories follow the below naming convention.
+
+`ffc-<workstream>-<service>`
+
+For example, `ffc-elm-apply`.
+
+Following this naming convention helps understand ownership of repositories and is also essential for automated secret scanning managed in Jenkins.
+
+### Managing access
+The [FFC GitHub team](https://github.com/orgs/DEFRA/teams/ffc) is added with `Admin` access to all repositories hosted on GitHub. For AzureDevOps repositories, the equivalent `FFC` team should be used.
+
+As per [Defra policy](https://github.com/DEFRA/dst-guides/tree/master/github), contractors should be added as contributors and should not be part of the FFC Team in GitHub.
+
+### Branch policies
+- main branches are protected by a branch policy where merges requires a Pull Request approved by at least one other person.
+
+- stale reviews are automatically dismissed.
+
+- repositories only allow Squash merging.
+
+- as the FFC team is added in the `Admin` role, Administrators are included in policy rules.
+
+### Secret detection
+All developers are required to [install detect-secrets](../guides/developer-laptop-setup/install-detect-secrets.md)
+when working on open-source FFC git repositories in github.
+
+When setting up a new FFC git repository, it needs to be configured to work with the client-side `detect-secrets` tool:
+
+1. Add the configuration file called `.pre-commit-config.yaml` to the repository root directory with the following contents:
+
+```
+repos:
+- repo: https://github.com/Yelp/detect-secrets
+  rev: v0.13.1
+  hooks:
+  - id: detect-secrets
+    args: ['--baseline', '.secrets.baseline']
+```
+
+2. Create and add the baseline file. To exclude files from the `detect-secrets` scan, run with `--exclude-files <regex>`. Examples:
+
+```
+detect-secrets scan > .secrets.baseline
+```
+
+```
+detect-secrets scan --exclude-files package-lock.json > .secrets.baseline
+```
+
+### Licence
```

Comment:
Is it worth having this as a file that can be copied into the repo?

Reply:
Good idea, I've added a `resources` folder and added it there.